Snippets/Snippets_5/Monitoring/EndpointConfig.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace MonitoringNotifications.ServiceControl.Notifications
+{
+    using System;
+    using NServiceBus;
+    using NServiceBus.Persistence;
+
+    #region MessageFailedEndpointConfig 5.0
+    public class EndpointConfig : IConfigureThisEndpoint, AsA_Server
```

Comment:
>It's a minor pitfall, if overlooked the whole thing will fail...

But you havent actually said that. How is a person meant to know that `JsonSerializer` is important but `RavenDBPersistence` isnt. 

Also the point i was trying to make was that using the host concept of `IConfigureThisEndpoint`  when u just want to illustrate the config api is overkill.